#+title: Understanding host-pathogen interactions and (co)evolution through coalescent genealogies and ancestral recombination graphs
#+subtitle: Linking epidemiological dynamics, (co)evolution and genomic ancestry of host-pathogen interactions
#+setupfile: ~/.doom.d/setup-files/latex.setup
#+OPTIONS: toc:nil author:nil

#+LATEX_HEADER: \author[1]{Philip L. Wolper}
#+LATEX_HEADER: \affil[1]{Chair of Population Genetics, Technical University of Munich, Germany}

#+LATEX_HEADER: \addbibresource{~/biblio/pop-gen.bib}

#+LATEX_HEADER: \usepackage{tabularx}

* Introduction
Host-pathogen coevolutionary systems are shaped by a complex interplay of epidemiological, ecological and evolutionary processes. Disease transmission, recovery and population dynamics of hosts and pathogens interact with mutation, selection, recombination and genetic drift to generate feedbacks that operate across  multiple timescales. Understanding how these processes jointly shape genomic variation and the genealogies (i.e evolutionary ancestral trees) along the genomes of populations remains a central challenge for the fields of coevolution and molecular epidemiology, and requires theoretical frameworks that can integrate short-term epidemiological dynamics with long-term evolutionary processes. The following presents a short proposal introducing a PhD thesis, bginning to tacke these ideas.  

* Background and Motivation
** Epidemiology and (co)evolution across timescales
Epidemiological and evolutionary processes act on different (but overlapping) timescales, and are tightly linked in coevolving host-pathogen systems. The processes related to epidemiology (such as transmission, infection and recovery) typically operate on short timescales, shaping the population dynamics of infectious disease. On the other hand, evolutionary processes (such as mutation, recombination, selection and genetic drift) act on longer timescales and determine the evolutionary history of host and pathogen lineages over time. Importantly, these processes do not act independently from each other, as there is significant feedback between evolution and epidemiology: Epidemiological dynamics influence the evolution of hosts and pathogens, while evolutionary change can also shape future epidemic processes. Figure [[epi-evo-timescales]] shows how different processes shape both the epidemiology and evolution and how they interact with each other. 

When designing models it is especially important to keep the scale in mind at which the model operates. 
Epidemiological approaches alone miss things like selection for host and pathogen types and their genomic evolution including mutation and recombination. On the other hand, evolutionary models (without epidemiological dynamics) often have no, or simplyfying assumptions about population dynamics and epidemiology, being frequency-based models of selection and coevolutionary dynamics.

At a more fundamental level, in host–pathogen systems, epidemiological and evolutionary processes are highly dependent on each other, because processes affect each other at different levels. For example, transmission in an epidemiological setting is a determining factor shaping the assumptions pathogen reproduction in an evolutionary model. Transmission, together with other epidemiological factors such as host resistance dynamics determine the demographic dynamics of pathogens, resulting in non-linear population dynamics. At an evolutionary level, this demography influences population genetic process, such as through genetic drift, mutation, recomination and selection. Sadly, this means the constant-size, non-structured assumptions of most population genetics models (eg. Wright-Fisher model) are not fit descriptions of evolutionary epidemiology.
On top of transmission other determinants, such as immune genetics play a role in shaping the coevolutionary dynamics. The specificity of the interaction (as determined by the infection matrix) affects how population genetics quantities such as recombination will act will act in the hosts and pathogen. This is because recombination will only happen if distinct genotypes co-occur on the same host, called coinfection. The amount of coinfection will be determined by the spatial structure of transmission and dispersal as well as the infection matrix (and possibly strain-strain interactions). Thesse are just a few examples of the interplay of epidemiological processes and coevolutionary ones.

The correspondence between epidemiological and evolutionary processes implies that they both will affect the ancestral history of pathogens and their hosts. In pathogens infectious branching processes shape the pathogen genealogy, while in hosts the genealogy is shaped by resistance and reproductive success. Common signatures, such as the timing and shape of genealogiecal trees, distributions of common ancestors, and linkage along the genome will reflect information about transmission dynamics, host-pathogen interactions and coevolutionary history. As a result, genealogies offer a framework in which short-term infection dynamics and long-term evolutionary change can be studied jointly.

#+name: epi-evo-timescales
#+caption: Processes shaping epidemiology and evolution of hosts and pathogen and their relevance across different time and spatial scales. 
#+attr_latex: :width 1\linewidth  :center t :placement [t]
[[./figures/epidem_evolution_timescales.png]]

# #+attr_latex: :heigth 1.1\pageheight  :float sideways :center t :placement [h]
** Phylodynamics: connecting epidemiology and genealogies
Phylodynamics has been sucessful in linking trees and genealogies to epidemiological processes ([[cite:&featherstone-2022-epidem-infer]]). This sucess lies in a theoretical foundation, that links tree branching models (such as the coalescent or birth-death processes) to a wide range of common epidemiological compartmental models. Recent theoretical advances have formualated general colescent processes in the context of general epidemiological models with superspreading ([[cite:&didelot-2025-ances-proces]]).
Phylodynamics has allowed backwards inference of epidemiological parameters of observed pathogen genealogies through likelihood- or simulation-based inference methods (eg. ABC), and can be readily applied to data, such as viral or bacterial sequences. Methods can roughly be grouped into two types: parametric and non-parametric ([[cite:&featherstone-2022-epidem-infer]]). Parametric models assume that population parameters, are fixed functions in time, whereas non-parametric models allow for piecewise constant parameters over time (non-parametric phylodynamic models are also called skyline models. Phylodynamic methods have made use of the structured coalescent approaches to account for population structure in host and pathogen populations. In summary phylodynamics provides us with a theoretical framework to link genealogical trees of sample data to underlying population dynamics processes. 

** Host-pathogen coevolutionary theory
Coevolution refers to the reciprocal process of adaptations and counteradaptations between hosts and pathogens.
Coevolutionary models include a wider-range of population and quantitative genetics models and approaches, that describe the coevolving dynamics of hosts and pathogens, occuring at evolutionary time scales. Most commonly these models are designed to include host and pathogen genotypes, infection genetics, different types of frequency-dependent selection, as well and have readily been extended with mutation and recombination. As such coevolutionary models have been used to describe rich dynamics; including cyclical dynamics and maintainance of polymorphisms through trench warefare; and unstable equilibria leading to fixations through arms race dynamics, to name a few. 

The importance of including population dynamics in coevolutionary models is recognized, as eco-evolutionary (epidemiological) feedbacks have a major impact on the model dynamics ([[cite:&buckingham-2022-coevol-theor]]). Yet still the literature of coevolutionary models is split by the inclusion or exclusion of population dynamics, and including population dynamics increases the complexity of the models significantly. Recently, forward-time simulation frameworks capable of tracking ancestral genealogies under recombination and complex population dynamics (e.g. SLiM 5.0) have been developed, enabling the study of ancestry patterns generated by realistic coevolutionary models.

The genomic signatures arising in coevolutionary models are typically characterized with population-level summary statistics (eg. allele frequencies, diversity measures or linkage disequilibrium) (see [[cite:&tellier-2014-speed-adapt]], [[cite:&ebert-2020-host]]). While these approaches have yielded important insights into signatures of coevolutionary dynamics, the genealogical histories underlying recombining host and pathogen genomes are rarely modelled explicitly.  

** Genealogies and ancestral recombiantion graphs (ARGs) as descriptors of evolutionary history
Ancestral recombination graphs (ARGs) have emerged as a central object in modern population genetics, enabling the study of genome-wide ancestry in the presence of recombination and complex demography. Specifically the development of ways to represent, simulate and infer ARGs has lead to a conceptual shift in population genetics, from focusing on summary statistics to studying the underlying genealogical structures. As such ARGs can be interpreted as sequences of coalescent trees along the genome, as broken up by recombination. They connect fundamental evolutionary processes, such as drift, selection, recombination and demography, to observed genomic patters by making the full ancestral structure of the genome explicit. Coalescent theory provides the connection between sample genealogies and stochastic population processes that shape the underlying genealogies and evolutionary trees. 
Because ARGs offer this connection of observed genetic variation to evolutionary processes shaping coalescence and recombination of lineages, they are perfectly primed to be used for understanding the coevolutionary and epidemiology of hosts and pathogens.

Genealogical trees and graphs arise at different scales from epidemiological processes to evolutionary ones. Figure [[couple-genealogies]] shows how ARGs in hosts and pathogen in coevolutionary models from epidemiological assumptions.
At the epidemic timescale, following the infectious branching process of an epidemic results in a pathogen genealogy, which traces itself through infected hosts (Fig. [[couple-genealogies]]A). Considering the hosts also evolve under a stochastic population process, the pathogen genealogy will become embedded in a host genealogy, determined by host and pathogen reproduction and the infection  process that connects them (Fig. [[couple-genealogies]]B). We will refer to genealogies or ARGs that are coupled in this way as joint ARGs. Finally,  Both pathogen and host genomes undergo recombination, creating sequences of trees and ARGs (Fig. [[couple-genealogies]]C).

#+name: couple-genealogies
#+caption: Representations of genealogical trees and tree sequences from epidemiological to evolutionary settings. A) Transmission tree of infectious branching process (Figure adapted from [[cite:&kuehnert-2011-phylog-epidem]]). B) Pathogen genealogy is embedded in a host genealogy (Figure adapted from [[cite:&vienne-2013-cospec-vs]]). C) Pathogen and host genomes are recombining sequences of trees (ARGs) along both genomes.  
#+attr_latex: :width 1\linewidth  :center t :placement [t]
[[./figures/coupled_genealogies.png]]

Within this framework, ancestral recombination graphs can be used in several distinct ways, prompting different questions: ARGs as /explanatory/ objects (How does epidemiology and coevolution shape ARGs?); ARGs as /descriptive/ summaries (What information do genealogies encode about the epidemiological and coevolutionary processes?); ARGs as /inference/ targets (Can we reconstruct ARGs robustly under realistic population dynamics and coevolutionary models?) As there is a duality between genealogies and genomes ([[cite:&ralph-2020-effic-summar]]), the ARGs will contain many classical genetic diversity statistics, as well as genetic linkage along the genome (see list of statistics in [[cite:&ebert-2020-host]]).

* Research Gap
/Limitations of phylodynamics for coevolving host-pathogen systems./ While phylodynamics is a young and rapidly advancing field, its applicability relies on several key assumptions. Most phylodynamic models are designed for rapidly evolving pathogens, such that mutations arise on the same timescale as transmission, and therefore are primarliy the inference is suited for short timescales, such as epidemic outbreaks. Additionally, most common epidemiological models, describe dynamics in terms of host compartments, with pathogen reproduction represented indirectly (eg. $N_e$ of infected host compartment) and host evolution is typically excluded.  While phylodynamics provides a powerful link between epidemiological models and pathogen phylogenies, they are fundamentally grounded in tree-based representations, that assume neglible recombination and do not describe evolutionary processes such as selection in a coevolutionary setting. This limits the applicability of phylodynamic methods to many coevolving biological systems, with complex reproductive life cycles and epidemiology. Adressing this gap requires moving beyond single trees towards genealogical representations that can accomodate recombination and long-term dynamics. 

/'Genealogical gap' in coevolutionary theory./ Despite their mechanistic richness, coevolutionary models currently lack a unifying genealogical and inferential framework comparable to phylodynamics. In particular, coevolutionary dynamics have not been analyzed within the framework of ancestral recombination graphs and genomic evolutionary trees, limiting our ability to link model dynamics to recombining genomic data. While many genomic sigantures of coevolution have been investigates (see list in [[cite:&ebert-2020-host]], [[cite:&tellier-2014-speed-adapt]]), these signatures have not been explained in the context of genealogies and coalescent tree-sequences.

 /Integrating Phylodynamics and Host-Pathogen Coevolution through Ancestral Recombination Graphs/. Taken together, current approaches leave a conceptual gap between phylodynamic models, which link genealogical trees to epidemiological processes on short timescales, and coevolutionary host–pathogen models, which describe long-term evolutionary dynamics (but remain largely disconnected from genealogical representations).  Ancestral recombination graphs (ARGs) offer a natural way to bridge this gap by extending tree-based phylodynamic frameworks to recombining genomes and by embedding epidemiological and coevolutionary processes directly into the structure of ancestry. With an increased ability to simulate ARGs under non-standard population genetic models, they have the potential to be used widely to study biological questions. However, how host–pathogen interactions, transmission dynamics, and recombination jointly shape ARGs (and reversly, what information ARGs can provide about underlying evolutionary and epidemiological processes) remains largely unexplored. Addressing this gap requires a unified framework that treats ARGs as the central objects linking epidemiology, coevolution, and genomic data.
 
* Research Questions and Aims
This project aims to establish ancestral recombination graphs (ARGs) as a unifying object for studying and inferring host-pathogen coevolution across epidemiological and evolutionary timescales. Our aims represent distinct but connected research questions involving explanatory theoretical modeling, inference from data and evaluation of existing methodological approaches. 

** Use ARGs for to describe and infer host-pathogen interactions
*Aim:* Use ARGs as descriptive and inferential objects resulting from host-pathogen coevolution and epidemiological dynamics. This involves setting up a realistic model to simulate host and pathogen ARGs, as well as using ARG-derived summary statistics to improve existing inference methods (eg. GxG ABC framework)

*Guiding Questions:*
- How do coevolution and epidemiology jointly shape ARGs of hosts and pathogens?
- Can ARG-derived statistics be used to infer genetic interactions, infection matrices, or epidemiological parameters from sampled data? 
- Can we extend tree-based phylodynamic inference to full ARGs?
- Do ARGs capture known genomic signatures of coevolutions and epidemiological dynamics?
- How can we apply coevolutionary models and inference to data beyond using simple summary statistics? 
  
**  Joint host-pathogen ARGs
*Aim:* To characterize the joint ARGs of hosts and their infecting pathogen samples and the role they could play in improving inference of host-pathogen interactions.

*Guiding questions:*
- What associations are there between the ARGs of hosts and pathogens, considering host samples are connected to their infecting pathogens?
- Do properties of joint ARGs encode evolutionary and epidemiological signals?
  
** ARG inference under complex coevolutionary dynamics
*Aim:* Test whether current ARG reconstruction methods can recover the true ARGs from data, considering biases introduced by complex coevolutionary and epidemiological models.
  
*Guiding questions:*
- What biases arise when reconstructing ARGs from sampled sequences under these models?
- How does recombination, selection and epidemiological demography affect the inference of ARGs?
- Which ARG-derived statistics are robust for downstream inference?

* Models and Methods :noexport:
- Sketch ARGs models, branching process models and coevolutionary models that can be used to describe host and pathogen ARGs. Simulation with SLiM.
- A joint ARG is a pair of host and pathogen ARGs whose branching, coalescence, and recombination events are statistically dependent through infection and transmission processes.
- GxG ABC inference outline.
- Data availability of crop and pathogen data. Barley-Fusarium and Ramularia.
  
* Course of action 
** Simulation framework
Develop a simulation framework to simulate joint host and pathogen ARGs with SLiM; This should include realistic epidemiology (transmission and infection); reproduction in hosts and pathogens; coevolutionary dynamcis (including costs, frequency-dependent selection and infection matrices); population structure; as well as population genetic processes, such as mutation and recombination.
** Use ARGs to improve simulation-based GxG inference  
Use ARGs and ARG-derived summaries, to adress the following limitations of current GxG inference framweworks based on approximate bayesian computation (ABC): 
  - Structured populations bias the inference of GxG and needs to be accounted for.
  - Simplified epidemiology in current ABC setup
  - Summary statistics used by ABC are sensitive to sample size and allele frequencies (eg. MAF filtering)
  - Computational scalability issues due to pairwise comparisons between SNPs. (Compare trees rather than SNPs...)

We first aim to improve the ability of the ABC to correctly discern between different models of coevolution using simulated data, and then we aim to apply the ABC framework to data from from Fusarium and Barley, with adjustments to our simulation framework to mirror the biological realities of the system.
** ARG-based inference of epidemiology
Use simulation to show how different epidemiological dynamics affect pathogen ARGs. Possibly extend simple phylodynamic inference to ARGs and apply to data.
  
* Bibliography
[[printbibliography:]]

