#+title: Inferring coevolutionary interaction from a coalescent perspective
#+author: Philip L. Wolper

#+setupfile: ~/.doom.d/setup-files/latex.setup
#+OPTIONS: toc:nil num:nil
#+LATEX_HEADER: \addbibresource{~/biblio/pop-gen.bib}



We want to investigate the joint behaviour of genealogies in coevolving species of host-parasite interactions. Describing the dynamics of coevolution at the level of genomic coalescence trees, our aims is to better understand the sigantures coevolution leaves in the genomes of interacting species to improve inference of both functional, quantitative GxG interactions, as well as population genetic information such as demographic and reproductive life-histories of both species.\\

** Questions and Aims
We want to investigate the joint behaviour of genealogies in coevolving species of host-parasite interactions. Describing the dynamics of coevolution at the level of genomic coalescence trees, our aims is to better understand the sigantures coevolution leaves in the genomes of interacting species to improve inference of both functional, quantitative GxG interactions, as well as population genetic information such as demographic and reproductive life-histories of both species.\\
*** How can we describe the co-dependence of host and pathogen genealogies?
As we know the pairings of hosts and pathogens (through time), can find a measure of correlations between the 'aligned' host and pathogen genealogies?  Above average appearance of host and parasite types together in pairwise nodes of the co-dependent genealogies? How will different statistical measures of the trees (eg. TMRCA and tree topology) be correlated in various co-evolutionary scenarios?  Relative importance of TMRCA vs. topology? Significant deviations from neutral 'co-evolutionary' signatures between indicate co-evolution (what does 'neutral' co-evolution mean?). The 'neutral' co-evolutionary model needs account for neutrality, linkage disequilibrium, life-history traits as well as demographic history.

*** Coalescent models used need to be descriptive of the co-evolutionary dynamics at play.
This might require coalescents other than Kingman's coalescent. Depending on life-history the host and pathogen might require different reproductive models, which give rise to the sample genealogy. For example, Wright-Fisher reproduction in host (or neutral agricultural sowing scheme) and sweepstakes reproduction in pathogens? Comparison of coalescent trees should be independent of coalescent model used to construct the sample genealogy, so to give freedom to model life-history and epidemiology. What role does the chosen epidemiological model have on the coalescent process used? Significance of multiple-merger coalescent in the spread of disease and epidemiological descriptions (see work on multifurcating skyline plot, [[cite:&hoscheit-2019-multif-skylin-plot]])? Can we connect the model of co-evolution to the multiple merger dynamics of skewed pathogen offspring distributions?  What is the relation of the forward (eg. disease dynamics) and backwards models (host and pathogen coalescents)? What information does the model of disease dynamics hold towards to description of coalescent genealogies of hosts and parasites?

*** Can co-dependent genealogies predict the interaction matrix of a genome-genome interaction?
Measure of correlations between genealogies between species should be able to predict the interaction matrix of a genome-genome interaction? Do different types of coalescent models fit to different scoevolutionary scenarios (eg. multiple merger to arms race, etc.)? Do different types co-dependent coalescent signatures fit to different co-evolutionary scenarios such as GFG/MA and how do they differ from the neutral case. Can we connect the epidemiological parameters of a pathogen to it's reproductive Canning's model/coalescent parameters? How do we choose coalescent model to use then? How do such coalescent behaviour reflect back on the epidemiologcal process of the pathogen?

*** How can this work be used improve co-evolutionary inference?
Improving the accuracy of the co-GWAS and the ABC computations to predict genomic regions of interactions? Extension of ABC statistics? We are looking to understand the genomic signatures of co-evolution from a comprehensive (tree-based) perspective, to improve existing inference of GxG interactions. Can we learn anything new about co-evolutionary interaction at the genomic level in the process? How can the coalescent-driven description of host-pathogen genealogies improve epidemiological modelling of diseases (eg. inferring more accurate reproductive parameters or dating transmission chains)?

** Connecting coalescent models to epidemiology
- Offspring distributions from epidemiologcal models: negative binomial ([[cite:&lloyd-smith-2005-super-effec]]) with the parameters $R_{0}$ and $k$.
- [[cite:&li-2017-quant-trans]] show that the dispersion paramter of the epidemiological offspring distribution $k$ can be estimated from the pathogen phylogeny, using a formulation of of the Kingman coalescent with arbitrary offspring distributions ($N_{e} = N/\sigma^{2}$,[[cite:&koelle-2011-rates-coales]], [[cite:&fraser-2017-coales]]). How valid is this assumption for MMC?
- Multifurcating skyline plot [[cite:&hoscheit-2019-multif-skylin-plot]] uses Beta-coalescent to infer the effective population size through time, and show that the estimation of \alpha correlates with the degree of superspreading $k$ in an epidemiological model. What is the direct connection between \alpha and $k$?

* Further thoughts
First, we need a basic model to describe the interaction of host and pathogen genealogies and study 'neutral' forces involved (i.e. constant population size, etc). How would we describe the infectivity/prevalence from the coalescent tree? Higher infectivity and increased transmission would result in a kind of offspring skew, because of higher fitness and more spores/offspring produced. Can we describe this offspring skew as a multiple merger genealogy. Then we are looking for local multiple merger signatures in the genome of pathogens, as a signal of a gene conferring a advantage contributing to it's increased spread (or any signal of positive selection in this sense). How do we know the selection is the result of coevolution (i.e. is associated with a variant in the host)? If this multiple merger siganture is still detectable in the pathogen genealogy, if we sample only pathogens found in the host carrying the variant, and not in the pathogens found in other variants, this means the selective advantage is only conferred by this particular host variant?

[[printbibliography:]]
