#+title: Coalescent-based inference of coevolution


* TODO Create dokument and summarize points from meeting with Aurelien
- connection of coevolution to coalescence trees. Can we predeict coevolutionary interactions from coalescent tree geneoalogies.

- First, we need a basic model to describe the interaction of host and pathogen genealogies and study 'neutral' forces involved (i.e. constant population size, etc). How would we describe the infectivity/prevalence from the coalescent tree? Higher infectivity and increased transmission would result in a kind of offspring skew, because of higher fitness and more spores/offspring produced. Can we describe this offspring skew as a multiple merger genealogy. Then we are looking for local multiple merger signatures in the genome of pathogens, as a signal of a gene conferring a advantage contributing to it's increased spread (or any signal of positive selection in this sense). How do we know the selection is the result of coevolution (i.e. is associated with a variant in the host)? If this multiple merger siganture is still detectable in the pathogen genealogy, if we sample only pathogens found in the host carrying the variant, and not in the pathogens found in other variants, this means the selective advantage is only conferred by this particular host variant?

- Do correlations of genealogies between species depend on the interaction matrix? One could imagine multiple-merger signatures from the repeated selective sweeps of arms-race coevolution to differe from the higher than average diversity found under balancing selection of trench-warfare.

- What kind of statistical moment can we find of coalescent trees, that describes both the TMRCA and their topology? Correlations between these trees can show coevolutionary signatures? Need to account for neutrality and linkage disequilibrium? Neutral life-history is important, as is demographic history.
